### PR TITLE
[Bugfix] Fix NPE when refresh mv does not exist

### LINKED ISSUE #9544: [sync materialized view]know error when refresh mv which not exists
### Steps to reproduce the behavior (Required)
REFRESH MATERIALIZED VIEW mv_not_exists;

### Expected behavior (Required)
`error:materialized view mv_not_exists not exists`
### Real behavior (Required)
```
2022-08-03 21:45:29,139 WARN (starrocks-mysql-nio-pool-259|4398) [StmtExecutor.execute():490] execute Exception, sql REFRESH MATERIALIZED VIEW mv_not_exists
java.lang.IllegalStateException: null
	at com.google.common.base.Preconditions.checkState(Preconditions.java:494) ~[spark-dpp-1.0.0.jar:?]
	at com.starrocks.sql.analyzer.MaterializedViewAnalyzer$MaterializedViewAnalyzerVisitor.visitRefreshMaterializedViewStatement(MaterializedViewAnalyzer.java:417) ~[starrocks-fe.jar:?]
	at com.starrocks.sql.analyzer.MaterializedViewAnalyzer$MaterializedViewAnalyzerVisitor.visitRefreshMaterializedViewStatement(MaterializedViewAnalyzer.java:62) ~[starrocks-fe.jar:?]
	at com.starrocks.sql.ast.RefreshMaterializedViewStatement.accept(RefreshMaterializedViewStatement.java:21) ~[starrocks-fe.jar:?]
	at com.starrocks.sql.ast.AstVisitor.visit(AstVisitor.java:113) ~[starrocks-fe.jar:?]
	at com.starrocks.sql.analyzer.MaterializedViewAnalyzer.analyze(MaterializedViewAnalyzer.java:59) ~[starrocks-fe.jar:?]
	at com.starrocks.sql.analyzer.Analyzer$AnalyzerVisitor.visitRefreshMaterializedViewStatement(Analyzer.java:293) ~[starrocks-fe.jar:?]
	at com.starrocks.sql.analyzer.Analyzer$AnalyzerVisitor.visitRefreshMaterializedViewStatement(Analyzer.java:76) ~[starrocks-fe.jar:?]
	at com.starrocks.sql.ast.RefreshMaterializedViewStatement.accept(RefreshMaterializedViewStatement.java:21) ~[starrocks-fe.jar:?]
	at com.starrocks.sql.ast.AstVisitor.visit(AstVisitor.java:113) ~[starrocks-fe.jar:?]
	at com.starrocks.sql.analyzer.Analyzer$AnalyzerVisitor.analyze(Analyzer.java:78) ~[starrocks-fe.jar:?]
	at com.starrocks.sql.analyzer.Analyzer.analyze(Analyzer.java:73) ~[starrocks-fe.jar:?]
	at com.starrocks.sql.StatementPlanner.plan(StatementPlanner.java:43) ~[starrocks-fe.jar:?]
	at com.starrocks.qe.StmtExecutor.execute(StmtExecutor.java:347) ~[starrocks-fe.jar:?]
	at com.starrocks.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:316) ~[starrocks-fe.jar:?]
	at com.starrocks.qe.ConnectProcessor.dispatch(ConnectProcessor.java:429) ~[starrocks-fe.jar:?]
	at com.starrocks.qe.ConnectProcessor.processOnce(ConnectProcessor.java:667) ~[starrocks-fe.jar:?]
	at com.starrocks.mysql.nio.ReadListener.lambda$handleEvent$0(ReadListener.java:55) ~[starrocks-fe.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_252]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_252]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_252]
```
### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`
```
mysql> select current_version();
+----------------------+
| current_version()    |
+----------------------+
| MAIN-RELEASE 12c19a2 |
+----------------------+
```

### PR DESCRIPTION
**Pull request by Astralidea**
## What type of PR is this：
- [x] bug
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #9544

## Problem Summary(Required) ：

At this time, because the materialized view does not exist when it is refreshed.

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 2 / 2 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java|2|2|100.00%|[]|

**Comment by Astralidea** [comment]
run starrocks_admit_test

File: fe/fe-core/src/test/java/com/starrocks/analysis/RefreshMaterializedViewStatementTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package com.starrocks.analysis;
```
Comment (imay) [inline fe/fe-core/src/test/java/com/starrocks/analysis/RefreshMaterializedViewStatementTest.java:1]: license header

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 2 / 2 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java|2|2|100.00%|[]|

**Comment by Astralidea** [comment]
run starrocks_admit_test

**Review by imay — APPROVED** [review]

**Review by ABingHuang — APPROVED** [review]
LGTM

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Review by gengjun-git — APPROVED** [review]

**Comment by Astralidea** [comment]
run starrocks_admit_test

**Pull request merged**